app/models/expense.rb

```diff
@@ -1,6 +1,13 @@
 class Expense < ActiveRecord::Base
   belongs_to :budget
 
-  validates :name, presence: true, uniqueness: true
-  validates :frequency, presence: true
+  enum frequency: [:Weekly, :Fortnightly, :Monthly, :Quarterly, :Annually]
```

Comment:
A suggestion to keep the symbol keys lowercase

Reply:
Done in sha: d4cc990